config/link-check/excluded-links.yml

```diff
@@ -18,3 +18,4 @@
 - 'https://timheuer.com/blog/skipping-ci-github-actions-workflows/'
 - 'https://www.zazzle.com/t_shirt-235920696568133954'
 - 'https://medium.com**'
+- 'https://twitter.com/flavioclesio/status/*'
```

Comment:
probably no need to include it if we are replacing it anyways

Reply:
We still have a link to it under the static tweet, which is what tripped the checker before the most recent commit

Reply:
my take on this - we should remove it completely (link is indeed broken, we should not have it in the content in the first place probably now)

Reply:
Yup! #3866 was opened for this. Right now it's keeping the quote, but we could remove the whole thing to be safe.